packages/xo-server/config.toml

```diff
@@ -137,7 +137,7 @@ proxyName = 'Proxy {date}'
 vmName = 'XOA Proxy {date}'
 
 # The duration for which we can wait for the VM networks to be defined
-vmNetworksTimeout = '1 min'
+vmNetworksTimeout = '5 min'
```

Comment:
Why?

Reply:
There have an intermediary IP given to the `VM` and this `IP` takes times to be updated.